ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/RemoteHostExecutor.java

```diff
@@ -143,4 +145,15 @@ public Pair<Map<HostInfo, KsqlEntity>, Set<HostInfo>> fetchAllRemoteResults() {
       executorService.shutdown();
     }
   }
+
+  private long getFetchRemoteHostsTimeout(){
+    final Object timeout
+        = executionContext.getKsqlConfig().getLong(KSQL_FETCH_REMOTE_HOSTS_TIMEOUT_SECONDS);
+
+    if (timeout instanceof Long) {
+      return (long) timeout;
+    }
```

Comment:
nit: I think `getLong` already returns `Long`?

Reply:
yeah but if the config is not set it will return null, which is why we check if it is indeed `Long`

Reply:
Will it use the default you defined if not set? `KSQL_FETCH_REMOTE_HOSTS_TIMEOUT_SECONDS_DEFAULT`

Reply:
I did not include it in the first commit of this pr and i got a bunch of test failures due to null exceptions when the config was not set like this 
https://jenkins.confluent.io/job/Confluent%20Public%20Repo%20PR%20builder/job/ksql/job/PR-9894/1/testReport/junit/io.confluent.ksql.rest.server.execution/ListQueriesExecutorTest/shouldListQueriesBasic/

I followed the pattern here for how to get the config/use the default if not set https://github.com/confluentinc/ksql/blob/19a8fcc67bdf582bd9bebe9778197e8238daf389/ksqldb-cli/src/main/java/io/confluent/ksql/cli/Cli.java#L455-L464

Reply:
That's weird. I saw `getLong` used like this: 

https://github.com/confluentinc/ksql/blob/master/ksqldb-engine/src/main/java/io/confluent/ksql/security/KsqlCacheAccessValidator.java#L118-L119

Reply:
Nvm you are right, the tests were just broken, i removed the `getFetchRemoteHostsTimeout` function and fixed the tests to avoid the null pointer exceptions, good catch thank you!